service/src/main/java/com/fit_goal/service/NotificatorService.java

```diff
@@ -0,0 +1,58 @@
+package com.fit_goal.service;
+
+import com.fit_goal.Audit;
+import com.fit_goal.api.Notificator;
+import com.fit_goal.domain.AuditDto;
+import com.fit_goal.domain.UserVerification;
+import com.fit_goal.enums.Notification;
+import com.fit_goal.util.MailSender;
+
+import javax.inject.Inject;
+import java.time.LocalDateTime;
+
+public class NotificatorService implements Notificator {
```

Comment:
Same here NotificationService and NotificationServiceImpl, that's de facto a standart from the Spring world

Reply:
refactored